### PR TITLE
sbdi-gtdb=R08-RS214-1 (r214)

### PR DESCRIPTION
**Pull request by erikrikarddaniel**
## PR checklist

- [x] This comment contains a description of changes (with reason).
- [ ] If you've fixed a bug or added code that should be tested, add tests!
- [ ] If you've added a new tool - have you followed the pipeline conventions in the [contribution docs](https://github.com/nf-core/ampliseq/tree/master/.github/CONTRIBUTING.md)
- [ ] If necessary, also make a PR on the nf-core/ampliseq _branch_ on the [nf-core/test-datasets](https://github.com/nf-core/test-datasets) repository.
- [ ] Make sure your code lints (`nf-core lint`).
- [ ] Ensure the test suite passes (`nf-test test main.nf.test -profile test,docker`).
- [ ] Check for unexpected warnings in debug mode (`nextflow run . -profile debug,test,docker --outdir <OUTDIR>`).
- [ ] Usage Documentation in `docs/usage.md` is updated.
- [ ] Output Documentation in `docs/output.md` is updated.
- [x] `CHANGELOG.md` is updated.
- [ ] `README.md` is updated (including new tool citations and authors/contributors).

Finally, last year's release of GTDB in SBDI-GTDB vetted format.

### CODE REVIEW

File: conf/ref_databases.config [hunk 1]
```diff
@@ -130,6 +130,14 @@ params {
             dbversion = "SBDI-GTDB-R07-RS207-1 (https://scilifelab.figshare.com/articles/dataset/SBDI_Sativa_curated_16S_GTDB_database/14869077/4)"
```
Comment (d4straub) [inline conf/ref_databases.config:130]: Key `'sbdi-gtdb'` still leads to Release `R07-RS207-1`, wouldnt it be good to update it?

**Review by d4straub — APPROVED** [review]
LGTM, just one comment.
Also, GTDB release 220 came out in April, maybe we should add that as well

File: conf/ref_databases.config (hunk 1 above)
Comment (erikrikarddaniel) [reply conf/ref_databases.config]: Good catch! I've pushed an update.

**Comment by erikrikarddaniel** [comment]
> LGTM, just one comment. Also, GTDB release 220 came out in April, maybe we should add that as well

Coming up. I've run my pipeline and produced the files. Now this needs to be submitted for storage and I'm not sure I'll make it before going on vacation next week.

**Comment by d4straub** [comment]
Several tests use that file, so I assume that was a temporal problem and resumed the test. Hope it will pass ;)

**Pull request merged**